templates/kubernetes/what-is-kubernetes.html

```diff
@@ -598,21 +594,17 @@ <h3>
               attrs={"class": "p-heading-icon__img p-heading-icon__img--small"},
               ) | safe
             }}
-            <h4 class="p-heading-icon__title">Data sheet</h4>
+            <h4 class="p-heading-icon__title" style="color: #fff;">Data sheet</h4>
```

Comment:
We've recently change the design of these content strips, take a look at this [PR](https://github.com/canonical-web-and-design/anbox-cloud.io/pull/196/files), it links to the issue where the new design was suggested.
- You'll need to add `is-dark` class to the `<div class="row p-divider">` this will keep all the text and divider lines lighter, so you won't need the inline styles. 
- Add `p-heading--4` class to line 597 and put it in a `<h3>` tag as at the moment it's not in the correct order of [headings](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements) - this will make it look like a h4 but keep it semantically correct